{"uid":"47477bc032c19156","name":"User logs in with valid credentials","fullName":"login functionality: User logs in with valid credentials","historyId":"92e0ff42c53e910eea45d3ad5eccc404","time":{"start":1728969377625,"stop":1728969408440,"duration":30815},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//h4\"}\n  (Session info: chrome=129.0.6668.90)\nStacktrace:\n\tGetHandleVerifier [0x00EB5523+24195]\n\t(No symbol) [0x00E4AA04]\n\t(No symbol) [0x00D42093]\n\t(No symbol) [0x00D86ED2]\n\t(No symbol) [0x00D8711B]\n\t(No symbol) [0x00DC76F2]\n\t(No symbol) [0x00DAAB84]\n\t(No symbol) [0x00DC5280]\n\t(No symbol) [0x00DAA8D6]\n\t(No symbol) [0x00D7BA27]\n\t(No symbol) [0x00D7C43D]\n\tGetHandleVerifier [0x0117CE13+2938739]\n\tGetHandleVerifier [0x011CEC69+3274185]\n\tGetHandleVerifier [0x00F409C2+594722]\n\tGetHandleVerifier [0x00F47EDC+624700]\n\t(No symbol) [0x00E537CD]\n\t(No symbol) [0x00E50528]\n\t(No symbol) [0x00E506C5]\n\t(No symbol) [0x00E42CA6]\n\tBaseThreadInitThunk [0x75EFFCC9+25]\n\tRtlGetAppContainerNamedObjectPath [0x77887C6E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77887C3E+238]\n\n","statusTrace":"  File \"C:\\Python310\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python310\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_login.py\", line 43, in step_impl\n    assert \"Dashboard\" == context.driver.find_element(By.XPATH, \"//h4\").text\n  File \"C:\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 856, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"C:\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 434, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"smoke"},{"name":"feature","value":"login functionality"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["smoke"]},"source":"47477bc032c19156.json","parameterValues":[]}